documentation/docs/pages/operators/variables.mdx

```diff
@@ -0,0 +1,19 @@
+{/*
```

Comment:
I'm not seeing the table in the rendered page - is that because of this comment?

Reply:
This page needs to be finished after merge when we implement cmdrun alternative to operators, I will make a new fork for that